scripts/calculate_percentile_offline.py

```diff
@@ -37,13 +39,14 @@
 
 # the algorithm
 def main():
-    parse_weather_dates()
-    remove_data_outside_date_range()
-    remove_data_outside_fire_season()
-    sort_by_weather_station()
-    calculate_percentile_per_station()
-    list_years_per_station()
-    write_output_to_json()
+    for item in range(len(RANGES)):
```

Comment:
Should be able to do
```suggestion
    for item in RANGES:
```